doc/compliancetest.rst

```diff
@@ -324,7 +346,7 @@ For multi-port devices:
 For conformance class B:
 
 * Topology discovery check
-* Non-Profinet neighbour setup
+* Topology non-PN  (Use non-Profinet-neighbor setup)
```

Comment:
We use British English spelling. Neighbour.

Reply:
Done